app.alfajores.yaml

```diff
@@ -1,4 +1,4 @@
-runtime: nodejs12
+runtime: nodejs18
```

Comment:
😱 yay!!

Reply:
i've noticed the indexer deploys MUCH faster than IHL and I wonder if using a custom runtime in IHL is the difference